src/module.c

```diff
@@ -12210,6 +12210,76 @@ int RM_LoadConfigs(RedisModuleCtx *ctx) {
     return REDISMODULE_OK;
 }
 
+int RM_RdbLoad(const char *filename, int flags) {
+    errno = 0;
+
+    if (flags != 0) {
+        errno = EINVAL;
+        return REDISMODULE_ERR;
+    }
+
+    /* TODO:
+     * - Call fsync for filename
+     * - Stop AOF?
+     * - Kill RDB fork?
+     */
+    emptyData(-1,EMPTYDB_NO_FLAGS,NULL);
+
+    if (server.current_client)
+        protectClient(server.current_client);
+
+    int ret = rdbLoad((char*) filename,NULL,RDBFLAGS_NONE);
+
+    if (server.current_client)
+        unprotectClient(server.current_client);
+
+    /* TODO:
+     * - rename(filename, server.rdb_filename) ?;
+     * - Start AOF?
+     */
+
+    if (ret == RDB_NOT_EXIST) {
+        errno = ENOENT;
+        return REDISMODULE_ERR;
+    } else if (ret == RDB_FAILED) {
+        errno = EIO;
+        return REDISMODULE_ERR;
+    }
+
+    serverAssert(ret == RDB_OK);
+    errno = 0;
+    return REDISMODULE_OK;
+}
+
+int RM_RdbSave(const char *filename, int flags) {
+    errno = 0;
+
+    if (!filename || flags != 0) {
+        errno = EINVAL;
+        return REDISMODULE_ERR;
+    }
+
+    if (server.child_type == CHILD_TYPE_RDB) {
+        errno = EINPROGRESS;
+        return REDISMODULE_ERR;
+    }
+
+    server.stat_rdb_saves++;
```

Comment:
i don't think we wanna update stats, we don't know where this file is saved to, from my perspective, it's not part of redis's persistence mechanism.

Reply:
ok, removed